src/identitylookup.cs

```diff
@@ -215,4 +242,31 @@ interface ILMessageCommunication {
 		[Export ("isEqualToMessageCommunication:")]
 		bool IsEqualTo (ILMessageCommunication communication);
 	}
+
+	[NoWatch, NoTV, NoMac, MacCatalyst (16,0), iOS (16,0)]
+	[Protocol]
+	interface ILMessageFilterCapabilitiesQueryHandling
+	{
+		[Async]
+		[Abstract]
+		[Export ("handleCapabilitiesQueryRequest:context:completion:")]
+		void HandleQueryRequest (ILMessageFilterCapabilitiesQueryRequest capabilitiesQueryRequest, ILMessageFilterExtensionContext context, Action<ILMessageFilterCapabilitiesQueryResponse> completion);
+	}
+
+	[NoWatch, NoTV, NoMac, MacCatalyst (16,0), iOS (16,0)]
+	[BaseType (typeof (NSObject))]
+	interface ILMessageFilterCapabilitiesQueryResponse : NSSecureCoding
+	{
+		[Export ("transactionalSubActions", ArgumentSemantic.Copy)]
+		NSNumber[] TransactionalSubActions { get; set; }
+
+		[Export ("promotionalSubActions", ArgumentSemantic.Copy)]
+		NSNumber[] PromotionalSubActions { get; set; }
```

Comment:
Both of these properties according to [documentation](https://developer.apple.com/documentation/sms_and_call_reporting/ilmessagefiltercapabilitiesqueryresponse) are `ILMessageFilterSubAction []`

```suggestion
		[BindAs (typeof (ILMessageFilterSubAction []))]
		[Export ("transactionalSubActions", ArgumentSemantic.Copy)]
		NSNumber[] TransactionalSubActions { get; set; }

		[BindAs (typeof (ILMessageFilterSubAction []))]
		[Export ("promotionalSubActions", ArgumentSemantic.Copy)]
		NSNumber[] PromotionalSubActions { get; set; }
```

Reply:
Ah good catch!